src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -442,7 +442,10 @@
         action.setCallback(this, function (response) {
             var state = response.getState();
             if (state === 'SUCCESS') {
-                this.openBatchWizard(component, event);
+                if (!component.get('v.modalOpen')) {
```

Comment:
This comment arguably falls outside of this work, but we recently did some refactor to similar code in BGE_BatchGiftEntryTabHelper.js and I wonder if it's worth bringing the same update in here as long as we are opening the hood - if you look at the "openNewBatchWizard" and "checkFieldPermissions" functions in there, we switched to passing in the 'onSuccess' behavior as an argument to this method, in an effort to avoid hard-coding the wizard open behavior as an outcome of checking the field permissions. 

Let me know if that all makes sense!

Reply:
Got it. Checking that out now